CHANGELOG/CHANGELOG-1.33.md

```diff
@@ -614,7 +614,7 @@ name | architectures
 
 ### API Change
 
-- A new status field `.status.terminatingReplicas` is added to Deployments and ReplicaSets to allow tracking of terminating pods when the DeploymentPodReplacementPolicy feature-gate is enabled. ([#128546](https://github.com/kubernetes/kubernetes/pull/128546), [@atiratree](https://github.com/atiratree)) [SIG API Machinery, Apps and Testing]
+- A new status field `.status.terminatingReplicas` is added to Deployments and ReplicaSets to allow tracking of terminating pods when the DeploymentReplicaSetTerminatingReplicas feature-gate is enabled. ([#128546](https://github.com/kubernetes/kubernetes/pull/128546), [@atiratree](https://github.com/atiratree)) [SIG API Machinery, Apps and Testing]
```

Comment:
I don't think we'd change for 1.33, so revert this change

Reply:
Ok, I suppose, we can change it in the next release, but it will be confusing a bit.

Reply:
Btw, this is already documented https://github.com/kubernetes/website/pull/49896